src/pages/2022-gartner-cool-vendors-in-software-engineering.tsx

```diff
@@ -0,0 +1,53 @@
+import { FunctionComponent } from 'react'
+
+import { Layout, SimpleGatedPgLayout } from '@components'
+
+export const Report: FunctionComponent = () => (
+    <Layout
+        meta={{
+            title: 'Cool Vendors in Software Engineering: Enhancing Developer Productivity',
+            description:
+                'Get free access to Gartner’s 2022 Cool Vendor Report to learn how Sourcegraph and other innovative tools help engineers and engineering leaders boost developer productivity while mitigating security risks.',
+        }}
+        className="navbar-white"
+    >
+        <SimpleGatedPgLayout
+            title="Cool Vendors in Software Engineering: Enhancing Developer Productivity"
+            customer={{
+                name: 'Gartner',
+                logo: '/external-logos/gartner-cool-vendor-2022.svg',
+            }}
+            formLabel="Download the report"
+            form={{
+                formId: '4f5b127f-4d42-490a-a970-686574d0e542',
+                onFormSubmitted: () =>
+                    window.open('https://www.gartner.com/doc/reprints?id=1-2A44UTV8&ct=220524&st=sb'),
+            }}
+            description={
+                <section className="col-md-6 col-12 pr-lg-6">
+                    <h3 className="font-weight-normal pb-5">
+                        Get complimentary access to 2022 Gartner® Cool Vendors™ report to learn how we believe
```

Comment:
Side note: The screen reader announced the special characters correctly 👌 I was curious if they would without the encoding (like `&#8482;`). TIL that the UTF-8 character is preferred for readability, like you did here! Issues can happen when the font has insufficient coverage, but this isn't the case here.

Reply:
Woohoo!